docs/service_integration.md

```diff
@@ -0,0 +1,46 @@
+# SALMON New Service Integration
```

Comment:
I would also consider slightest changes in wording (in .md file name and inside the doc).
like "adding_aws_service_integration.md"
And "AWS" from my perspective might be important.